Source/WTF/wtf/FunctionDispatcher.h

```diff
@@ -45,8 +45,9 @@ class WTF_EXPORT_PRIVATE FunctionDispatcher {
 
 class WTF_CAPABILITY("is current") WTF_EXPORT_PRIVATE SerialFunctionDispatcher : public FunctionDispatcher {
 public:
+    virtual bool isCurrent() const = 0;
 #if ASSERT_ENABLED
-    virtual void assertIsCurrent() const = 0;
+    void assertIsCurrent() const;
```

Comment:
Here now you can remove the assetIsCurrent method.

The intended contract is 
```
 assertIsCurrent([something that can be current]);
```

You'd then add the intended overload if none of the current overloads match.

Reply:
indeed. thanks